python/src/aac/plugins/validators/_validator_result.py

```diff
@@ -34,6 +33,6 @@ def get_messages_as_string(self) -> str:
         """Get all of the validator result messages as a single string."""
 
         def format_message(finding: ValidatorFinding) -> str:
-            return f"\nValidation finding from '{finding.location.validation_name}' of level {finding.severity.name} in {finding.location.source.uri}. Message:\n {finding.message}"
+            return f"\nValidation finding from '{finding.location.validation_name}' of level {finding.severity.name} in {finding.location.source}. Message:\n {finding.message}"
```

Comment:
Shouldn't we keep the `.uri` here? Otherwise, it will make the output much harder to read (see below...)

```
Validation finding from 'Type references exist' of level ERROR in AaCFile(uri='/file/path/aac/python/model/alarm_clock/alarm_clock.yaml', is_user_editable=True, is_loaded_in_context=False). Message:
 Undefined type 'Timetsamp' referenced: {'name': 'targetTime', 'type': 'Timetsamp', 'description': 'The target timestamp indicating when to fire the alarm alert.'}
```

instead of

```
Validation finding from 'Type references exist' of level ERROR in /file/path/aac/python/model/alarm_clock/alarm_clock.yaml. Message:
 Undefined type 'Timetsamp' referenced: {'name': 'targetTime', 'type': 'Timetsamp', 'description': 'The target timestamp indicating when to fire the alarm alert.'}
```

Reply:
Yeah, great catch!